examples_tests/39.DenoiserTonemapper/exampleInputArguments.txt

```diff
@@ -1,10 +1,2 @@
--OPENEXR_FILE=../../media/OpenEXR/daily_pt_256.exr -CHANNEL_NAMES=color,albedo,normal -CAMERA_TRANSFORM=1,0,0,0,0.83205,-0.5547,0,0.5547,0.83205 -EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_SIZE=1000,2000 -TONEMAPPER=ACES=2.51,0.03,2.43,0.59,0.14 -OUTPUT=out256_can.exr
--OPENEXR_FILE=../../media/OpenEXR/56_render_0_2_16.exr -CHANNEL_NAMES=color,albedo,normal -CAMERA_TRANSFORM=-0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_SIZE=1000,2000 -TONEMAPPER=ACES=2.51,0.03,2.43,0.59,0.14 -OUTPUT=56_render_0_2_16.exr
--OPENEXR_FILE=../../media/OpenEXR/56_render_0_2_32.exr -CHANNEL_NAMES=color,albedo,normal -CAMERA_TRANSFORM=-0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_SIZE=1000,2000 -TONEMAPPER=ACES=2.51,0.03,2.43,0.59,0.14 -OUTPUT=56_render_0_2_32.exr
--OPENEXR_FILE=../../media/OpenEXR/56_render_0_2_64.exr -CHANNEL_NAMES=color,albedo,normal -CAMERA_TRANSFORM=-0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_SIZE=1000,2000 -TONEMAPPER=ACES=2.51,0.03,2.43,0.59,0.14 -OUTPUT=56_render_0_2_64.exr
--OPENEXR_FILE=../../media/OpenEXR/56_render_0_2_128.exr -CHANNEL_NAMES=color,albedo,normal -CAMERA_TRANSFORM=-0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_SIZE=1000,2000 -TONEMAPPER=ACES=2.51,0.03,2.43,0.59,0.14 -OUTPUT=56_render_0_2_128.exr
--OPENEXR_FILE=../../media/OpenEXR/56_render_0_2_256.exr -CHANNEL_NAMES=color,albedo,normal -CAMERA_TRANSFORM=-0.733093,0.00218312,-0.680125,0,0.999995,0.00320987,-0.680129,-0.00235313,0.733089 -EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_SIZE=1000,2000 -TONEMAPPER=ACES=2.51,0.03,2.43,0.59,0.14 -OUTPUT=56_render_0_2_256.exr
\ No newline at end of file
+-COLOR_FILE=../../TEST.exr -CAMERA_TRANSFORM=1,0,0,0,0.83205,-0.5547,0,0.5547,0.83205 -MEDIAN_FILTER_RADIUS=0 -DENOISER_EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=0.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=1.0,0.08 -OUTPUT=out8_can.exr -ALBEDO_FILE=../../TEST2 -NORMAL_FILE=../../TEST3 -COLOR_CHANNEL_NAME=color -ALBEDO_CHANNEL_NAME=albedo -NORMAL_CHANNEL_NAME=normal -BLOOM_PSF_FILE=../TEEST
+-COLOR_FILE=../../TEcccST.exr -CAMERA_TRANSFORM=1,0,0,0,0.83205,-0.5547,0,0.5547,0.83205 -MEDIAN_FILTER_RADIUS=0 -DENOISER_EXPOSURE_BIAS=-2.0 -DENOISER_BLEND_FACTOR=3.0 -BLOOM_FOV=0.5 -TONEMAPPER=ACES=1.0,0.08 -OUTPUT=odut8_can.exr -ALBEDO_FILE=../../TEST2 -NORMAL_FILE=../../TEST3 -COLOR_CHANNEL_NAME=colo2r -ALBEDO_CHANNEL_NAME=alsbedo -NORMAL_CHANNEL_NAME=normal -BLOOM_PSF_FILE=../TEEsST
```

Comment:
all files need to have proper extensions on the input

Reply:
also rewrite with the old files so you can run and test them (use same input files)

Reply:
done